include/BLE_Common.h

```diff
@@ -36,9 +36,12 @@
 #define FITNESSMACHINEPOWERRANGE_UUID BLEUUID((uint16_t)0x2AD8)
 
 // GATT service/characteristic UUIDs for Flywheel Bike from ptx2/gymnasticon/
-#define FLYWHEEL_UART_SERVICE_UUID BLEUUID("6e400001b5a3f393e0a9e50e24dcca9e")
-#define FLYWHEEL_UART_RX_UUID BLEUUID("6e400002b5a3f393e0a9e50e24dcca9e")
-#define FLYWHEEL_UART_TX_UUID BLEUUID("6e400003b5a3f393e0a9e50e24dcca9e")
+#define FLYWHEEL_UART_SERVICE_UUID BLEUUID((uint16_t)0xCA9E)
+#define FLYWHEEL_UART_RX_UUID BLEUUID((uint16_t)0xCA9E)
+#define FLYWHEEL_UART_TX_UUID BLEUUID((uint16_t)0xCA9E)
+//#define FLYWHEEL_UART_SERVICE_UUID BLEUUID("6e400001b5a3f393e0a9e50e24dcca9e")
```

Comment:
Only because they are untested in the new form. I'm not familiar enough with the conversion to the short form to know for 100% they are correct until I get a report back from a Flywheel bike owner.

Reply:
Got it